src/components/views/dialogs/ServerPickerDialog.tsx

```diff
@@ -217,6 +217,7 @@ export default class ServerPickerDialog extends React.PureComponent<IProps, ISta
                         value={this.state.otherHomeserver}
                         validateOnChange={false}
                         validateOnFocus={false}
+                        id="homeserverInput"
```

Comment:
I think we typically prefix identifiers with `mx_`

Reply:
Good point
```suggestion
                        id="mx_homeserverInput"
```